bedita-app/controllers/components/api_validator.php

```diff
@@ -597,8 +608,11 @@ public function checkRelations(array $relations, $objectType = null) {
                 if (!$this->isRelationValid($inverseName, $relatedObjectType)) {
                     throw new BeditaBadRequestException('Invalid relation: ' . $name . ' for object type ' . $relatedObjectType);
                 }
-                if (!$this->isObjectReachable($relData['related_id'])) {
-                    throw new BeditaBadRequestException('Invalid Relation: ' . $relData['related_id'] . ' is unreachable');
+
+                if (empty($this->reachableRelations) || !in_array($name, $this->reachableRelations)) {
```

Comment:
Since `$this->reachableRelations` is an array maybe we should simplify as

```php
if (!in_array($name, $this->reachableRelations) && !$this->isObjectReachable($relData['related_id'])) {
    throw new BeditaBadRequestException('Invalid Relation: ' . $relData['related_id'] . ' is unreachable');
}
```